mobile/native/src/db/models.rs

```diff
@@ -637,6 +638,7 @@ pub enum FailureReason {
     ProposeDlcChannel,
     OrderNotAcceptable,
     TimedOut,
+    InvalidDlcOfferOfferOutdated,
```

Comment:
🔧 I think `SubchannelOfferOutdated` would be clearer i.e. without the `Invalid` in front and with `Subchannel` instead of `Dlc`.